test/pg_handler/start

```diff
@@ -16,3 +16,12 @@ docker-compose up \
   secretless
 
 sleep 2
+
+cat > .env <<EOF
```

Comment:
If this is only used locally, you should probably add it to dockerignore to avoid cache busting.